core/src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -62,36 +64,40 @@ object ManagedGroup {
-                               servicePerimeter: Option[ServicePerimeterName] = None,
-                               googleProjectNumber: Option[GoogleProjectNumber] = None,
-                               invalidBillingAccount: Boolean = false,
-                               spendReportDataset: Option[BigQueryDatasetName] = None,
-                               spendReportTable: Option[BigQueryTableName] = None,
-                               spendReportDatasetGoogleProject: Option[GoogleProject] = None,
-                               azureManagedAppCoordinates: Option[AzureManagedAppCoordinates] = None,
-                               billingProfileId: Option[String] = None
+case class RawlsBillingProject(
+  projectName: RawlsBillingProjectName,
+  status: CreationStatuses.CreationStatus,
+  billingAccount: Option[RawlsBillingAccountName],
+  message: Option[String],
+  cromwellBackend: Option[CromwellBackend] = None,
+  servicePerimeter: Option[ServicePerimeterName] = None,
+  googleProjectNumber: Option[GoogleProjectNumber] = None,
+  invalidBillingAccount: Boolean = false,
+  spendReportDataset: Option[BigQueryDatasetName] = None,
+  spendReportTable: Option[BigQueryTableName] = None,
+  spendReportDatasetGoogleProject: Option[GoogleProject] = None,
+  azureManagedAppCoordinates: Option[AzureManagedAppCoordinates] = None,
+  billingProfileId: Option[String] = None
 ) {
   // def instead of val because val confuses the json formatter
   def googleProjectId: GoogleProjectId = GoogleProjectId(projectName.value)
 }
 
-case class WorkspaceBillingAccount(workspaceName: WorkspaceName,
-                                   currentBillingAccountOnGoogleProject: Option[RawlsBillingAccountName]
+case class WorkspaceBillingAccount(
+  workspaceName: WorkspaceName,
+  currentBillingAccountOnGoogleProject: Option[RawlsBillingAccountName]
 )
 
-case class RawlsBillingProjectResponse(projectName: RawlsBillingProjectName,
-                                       billingAccount: Option[RawlsBillingAccountName],
-                                       servicePerimeter: Option[ServicePerimeterName],
-                                       invalidBillingAccount: Boolean,
-                                       roles: Set[ProjectRoles.ProjectRole],
-                                       status: CreationStatuses.CreationStatus,
-                                       message: Option[String],
-                                       managedAppCoordinates: Option[AzureManagedAppCoordinates]
+case class RawlsBillingProjectResponse(
+  projectName: RawlsBillingProjectName,
+  billingAccount: Option[RawlsBillingAccountName],
+  servicePerimeter: Option[ServicePerimeterName],
+  invalidBillingAccount: Boolean,
+  roles: Set[ProjectRoles.ProjectRole],
+  status: CreationStatuses.CreationStatus,
+  message: Option[String],
+  managedAppCoordinates: Option[AzureManagedAppCoordinates], // TODO: do we even want to return this?
```

Comment:
We do wish to remove the managedAppCoordinates, but terra-ui will need to be updated first to use the cloud context instead. See https://github.com/broadinstitute/rawls/pull/1994#discussion_r949397934

We probably don't want to check in the code with a TODO.